### DISCUSSION TITLE
Removal of inline typed dict syntax

### DISCUSSION THREAD

**Discussion by Gobot1234** [microsoft/pyright#5682] [General]
Curious as to whether the provisional support for this was removed, it seems to have stopped working in my project <img width="689" alt="image" src="https://github.com/microsoft/pyright/assets/50501825/01168eb2-af47-4402-b7cd-fc03dcce41e4">
but I can still see that createTypedDictTypeInlined still exists. I also can't see anything in the commit history saying it's been removed.

**Comment by erictraut** [comment]
It was moved behind a configuration flag called `enableExperimentalFeatures`. This feature is likely to be removed altogether in the near future because no one has stepped forward to push it forward as a standard feature. If you're using it in any of your production code, I recommend removing its use.
